src/components/Navigator/NavigatorCardItem.vue

```diff
@@ -176,6 +195,12 @@ export default {
         this.selfFocus();
       }
     },
+    async expanded() {
+      // wait a few frames for animations queues to finish
+      await waitFrames(8);
```

Comment:
I'm guessing there isn't, but is there possibly a more reliable way of doing this as opposed to waiting for 8 frames?

If not, maybe it should actually be 9 since the the animation duration is 0.15 seconds and the refresh rate is probably most commonly 60hz (60 * .15 = 9). Even if my math is wrong, it might be good to add to the comment that the number of frames is largely estimated to match the duration of the CSS transition.

Reply:
Right, will do.

Reply:
Thanks for the math comment, I had no idea how to calculate those frames 😅